projects/storefrontlib/src/lib/storefront.module.ts

```diff
@@ -8,7 +8,8 @@ import {
   StateModule,
   SmartEditModule,
   CxApiModule,
-  I18nModule
+  I18nModule,
+  ProcessModule
```

Comment:
We have to find another way.  The store related code should not be directly imported/visible in the storefront lib.

Reply:
I agree, but without importing this into storefrontmodule the process state slice is not visible.

Reply:
We agreed to add the import in the facades modules